src/renderer/components/+landing-page/workspace-overview.tsx

```diff
@@ -0,0 +1,69 @@
+import "./workspace-overview.scss";
+
+import React, { Component } from "react";
+import { Workspace } from "../../../common/workspace-store";
+import { observer } from "mobx-react";
+import { ItemListLayout } from "../item-object-list/item-list-layout";
+import { ClusterItem, WorkspaceClusterStore } from "./workspace-cluster.store";
+import { navigate } from "../../navigation";
+import { clusterViewURL } from "../cluster-manager/cluster-view.route";
+import { WorkspaceClusterMenu } from "./workspace-cluster-menu";
+
+interface Props {
+  workspace: Workspace;
+}
+
+enum sortBy {
+    name = "name",
+    contextName = "contextName",
+    version = "version",
+}
+
+@observer
+export class WorkspaceOverview extends Component<Props> {
+
+  showCluster = (clusterItem: ClusterItem) => {
+    const clusterId = clusterItem.getId();
+
+    navigate(clusterViewURL({ params: { clusterId } }));
+  };
+
+  render() {
+    const { workspace } = this.props;
+    const workspaceClusterStore = new WorkspaceClusterStore(workspace.id);
+    
+    workspaceClusterStore.loadAll();
```

Comment:
The creation probably should be done in a field initialize. And the `loadAll` in `componentDidMount`?

Reply:
I had it that way at one point but it wasn't enough.   Need to update the store each time `render()` is called.

Reply:
That sounds very strange. Like the loading should be reactive to something else.